ivy_tests/test_docstrings.py

```diff
@@ -131,6 +232,13 @@ def check_docstring_examples_run(
         if ">>> print(" in line:
             is_multiline_executable = False
 
+    # remove "..." for multilines
+    for i, v in enumerate(executable_lines):
+        executable_lines[i] = v.replace("...", "")
```

Comment:
this is already been done at line [216](https://github.com/unifyai/ivy/pull/18158/files#diff-48ffb3a283811c82d46de1000e0dbf654f102f8fbb23f5dcf9c974e21964bc90R216) right? Probably there's some bug in that which could be why the replace isn't working

Reply:
```            executable_lines[-1] += line.split("...")[1][1:]```
This line removes first three dots but keep rest. I didn't change that part because it correctly extract multiline code that should be executed as single line.

Reply:
Just removed that part.